src/main/resources/org/bytedeco/javacpp/properties/linux-riscv64.properties

```diff
@@ -0,0 +1,31 @@
+platform=linux-riscv64
+platform.path.separator=:
+platform.source.suffix=.cpp
+platform.includepath.prefix=-I
+platform.includepath=
+platform.compiler=g++
```

Comment:
This needs to be to the compiler for linux-riscv64

Reply:
I added -march=rv64gc -mabi=lp64d for platform.compiler.default

Reply:
I don't think GCC supports multiple targets like that. You'll need to give it the correct binary name.

Reply:
platform.compiler should be riscv64-linux-gnu-g++  ,  however where should I  install riscv64 cross-compilation toolchain by the command  ' sudo apt install g++-riscv64-linux-gnu '  ,  could you tell me ? Thanks to you!

Reply:
It should already be installed, just give it a try

Reply:
It seems that https://github.com/bytedeco/javacpp-presets/pull/1544 does not work
![image](https://github.com/user-attachments/assets/326d2a2d-73fb-4d4d-b279-fe4c0a81c8e9)
![image](https://github.com/user-attachments/assets/6817dd18-7a53-49af-a255-885c1cd9f88c)

Reply:
Ok, I fixed that, but now we're getting this error:
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 openjdk-8-jdk : Depends: openjdk-8-jre (= 8u422-b05-1~20.04) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
That needs to be fixed

Reply:
Ok, I've updated the builds to Jammy and that seems to work now. So this looks good to merge, but could you please fix the formatting first? Thank you!